src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefUsersController.java

```diff
@@ -233,6 +238,12 @@
          * This API will need to be revisited if it is to be used for business functionality.
          */
 
+        log.info("ENVIRONMENT NAME:::::: " + environmentName);
+
+        if (environmentName.equalsIgnoreCase("PROD")) {
```

Comment:
Can you do the other way around? like
`"PROD". equalsIgnoreCase(environmentName)` so that we can avoid NPE if  by chance `environmentName ` is null

Reply:
Addressed here: https://github.com/hmcts/rd-caseworker-ref-api/pull/291/commits/f6d389a2124d37bc2f29eb85fb6fbc2fb1bdf972